.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Report an issue with existing supported devices.
+labels: ["bug"]
+body:
+  - type: input
+    id: info-device
+    attributes:
+      label: Affected Device
+      description: What is the name of the device you are using that is not functioning as expected? If more than one devices are experiencing issues list them here separated by a comma.
```

This is unnecessary long, IMO:

```suggestion
      description: Which device isn't functioning as expected?
```